pmc/index.json

```diff
@@ -11,7 +11,20 @@
     "dynamic": "strict",
     "properties": {
       "name": {
-        "type": "keyword"
+        "type": "keyword",
+        "index": true,
+        "doc_values": false,
+        "copy_to": ["name_docvalues", "name_indexordocvalues"]
```

Comment:
What is the reason the default for `doc_values` being changed for the `name` field?  Any additions to the workload should not modify the existing mapping.